### DISCUSSION TITLE
Is there a way to analyze only the commit files?

### DISCUSSION THREAD

**Discussion by CoolLoong** [JetBrains/Qodana#196] [Q&A]
I tried with `pr-mode: true`, but this doesn't seem to work on direct commits.

**Comment by brichbash** [comment]
Hello,

PR mode is enabled by default.  Do you want to analyze only changed files in PRs or always when pushed to your branch?
In this case you could try using the `--script local-changes` option. It can be set by adding to your Qodana job configuration: 
```
with:
  args: --script,local-changes
```
